src/DotLiquid/DotLiquid.csproj

```diff
@@ -28,7 +28,7 @@
     <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
-      <DocumentationFile>bin\Debug\$(TargetFramework)\DotLiquid.xml</DocumentationFile>
+    <DocumentationFile>bin\Debug\$(TargetFramework)\DotLiquid.xml</DocumentationFile>
```

Comment:
nit: what's this change?

Reply:
Format on save fixed the whitespace when I added, then removed the Json dependency.